conrad/cli.py

```diff
@@ -171,6 +171,12 @@ def _show(ctx, *args, **kwargs):
     events = list(
         session.query(Event).filter(*filters).order_by(Event.start_date).all()
     )
+    reminders = list(
```

Comment:
Instead of creating a list here, you can query the reminders table in the `for event in events` loop.

Reply:
updated